device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C28S4/BALANCED/buffers_defaults_t0.j2

```diff
@@ -0,0 +1,52 @@
+{%- set default_cable = '300m' %}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,32) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}

```

Comment:
Do we have the same port name mappings for 10G, like port_idx * 4? Please check.

Reply:
Good catch, we use the whole port, so it worked out to be the same port names.

Reply:
yes, I have confirmed that before. Since these 10G ports uses only 1 lane from the physical port, so the port number still jumps by 4.

Here is the port config ini:

![image](https://github.com/user-attachments/assets/0564fbda-e765-4838-968e-91db1fac53ec)